samples/gdf/README.md

```diff
@@ -0,0 +1,6 @@
+* To run the mnist_migraphx GDF, 
```

Comment:
please remove this readme, and amend the main readme in the /sample/readme.md to have these instructions there.